source/includes/fundamentals/code-snippets/bson-extra-elements/ProductWithBsonExtraElements.java

```diff
@@ -0,0 +1,75 @@
+package fundamentals;
+
+import java.util.List;
+
+import org.bson.BsonType;
+import org.bson.Document;
+import org.bson.codecs.pojo.annotations.BsonCreator;
+import org.bson.codecs.pojo.annotations.BsonDiscriminator;
+import org.bson.codecs.pojo.annotations.BsonExtraElements;
+import org.bson.codecs.pojo.annotations.BsonId;
+import org.bson.codecs.pojo.annotations.BsonIgnore;
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.bson.codecs.pojo.annotations.BsonRepresentation;
+
+@BsonDiscriminator(value="ProductWithBsonExtraElements", key="_cls")
```

Comment:
Is this annotation necessary the example?  If not, it should be removed

Reply:
Similar reason as mentioned in this comment https://github.com/mongodb/docs-java/pull/242#discussion_r939022903